src/boost/formatters.nim

```diff
@@ -64,21 +72,16 @@ proc alignStr*(s: string, len: int, fill = ' ', trunc = false): string =
   if len == 0:
     result = s
   elif trunc and s.len > abs(len):
+    result = s
```

Comment:
I would expect that truncation differentiates between a positive and a negative alignment (use cases: right-truncate is preferred for regular text, left-truncate can be useful for e.g. paths). That's why I added the test cases:

```nimrod
check: alignStr("abc", 2, trunc=true) == "ab"
check: alignStr("abc", -2, trunc=true) == "bc"
```